### PR TITLE
IA-2220 Improvements on CI

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-2220

Depends on https://github.com/broadinstitute/terra-helm/pull/156

### CODE REVIEW

File: core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ object DbReaderImplicits {
     s match {
       case "DATAPROC" => CloudService.Dataproc.asRight[String]
       case "GCE"      => CloudService.Gce.asRight[String]
-      case x          => s"invalid cloudService value ${x}".asLeft[CloudService]
+      case x          => s"invalid cloudService value $x".asLeft[CloudService]
```
Comment (Qi77Qi) [inline core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala:14]: this dummy change is just for trigger CI for updating both images in terra-helm once this is merged

File: .github/workflows/createPullRequest.yml [hunk 2]
```diff
@@ -25,13 +28,22 @@ jobs:
...
           repository: broadinstitute/terra-helm
+#          ref: ci  you can use this for testing a specific branch of terra-helm
       - name: update resource-validator hash
+        uses: awesome-global-contributions/action-yq@v0.1.2
         if: steps.resource-validator.outputs.changed == 'true'
-        run: .github/scripts/createCommit.sh resourceValidatorImageTag
-        shell: bash
+        with:
+          command: |
+            yq w -i charts/leonardo/values.yaml cronjob.imageTag ${{ steps.setHash.outputs.git_short_sha }}
```
Comment (rtitle) [inline .github/workflows/createPullRequest.yml:37]: TIL about `yq`

**Review by rtitle — APPROVED** [review]
Not super familiar with this stuff, but LGTM

**Comment by Qi77Qi** [comment]
@rtitle will talk about this more in walkthrough...definitely worth sharing!

**Pull request merged**